lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -187,6 +187,10 @@ data ApiWallet = ApiWallet
     , tip :: !ApiBlockReference
     } deriving (Eq, Generic, Show)
 
+newtype ApiWalletData = ApiWalletData
+    { passphrase :: ApiT (Passphrase "encryption")
+    } deriving (Eq, Generic, Show)
+
```

Comment:
What about 'ApiWalletPassphrase' to better reflect what it is?

Reply:
yes, it is even more precise. will correct in the coming commit